source/leuven/slurm-accounting.rst

```diff
@@ -3,58 +3,81 @@
-
-The accounting system is very similar to a regular bank. Individual users have accounts that will be charged for the jobs they run. However, the number of credits on such accounts is fairly small, so
-research projects will typically have one or more project accounts associated with them. Users that are project members can have their project-related jobs charged to such a project account. In this how-to,
-the technical aspects of accounting are explained.
+The :ref:`wICE <wice_t2_leuven>` cluster uses the Slurm Account Manager (SAM) 
+for credit accounting.
+The reason is that having a valid Slurm account with sufficient credits in it
+is mandatory for running jobs on KU Leuven Tier-2 machines. 
+We developed so called SAM-commands to mimic the behavior of previously used 
+MAM-commands (from the MOAB account manager). 
+From the user point of view it should mostly be enough to change the prefix ``mam`` 
+in the accounting command into ``sam``.
+
+The accounting system is very similar to a regular bank. 
+Individual users have accounts that will be charged for the jobs they run. 
+However, the number of credits on such accounts is fairly small, so research projects 
+will typically have one or more project accounts associated with them. 
+Users that are project members can have their project-related jobs charged to such a 
+project account. 
+In this how-to, the technical aspects of accounting are explained.
 
 Checking an account balance
 ---------------------------
 
-Since no calculations can be done without credits, it is quite useful to determine the amount of credits at your disposal. This can be done by executing the following command on any KU Leuven Tier-2 node::
+Since no calculations can be done without credits, it is quite useful to determine the 
+amount of credits at your disposal. 
+This can be done by executing the following command on any KU Leuven Tier-2 node::
 
    $ sam-balance
 
-This will provide an overview of the balance on the user's personal account, as well as on all project accounts the user has access to.
+This will provide an overview of the balance on the user's personal account, as well as 
+on all project accounts the user has access to.
 
 
 Running jobs: accounting workflow
 ---------------------------------
 
-When a job is submitted using ``sbatch`` or ``srun`` and it has to be charged against a project account, the name of the project has to be specified as an option. In case of the introduction
-credits the project account should be specified as  ``default_project``::
+When a job is submitted using ``sbatch`` or ``srun``, it has to be charged against a 
+project account.
+The name of the project has to be specified as an option. 
+If a user requests introductory credits, the assigned project name carries the VSC-id
+of the user, e.g. ``intro_vsc3....``, where the dots ``.`` signify the user's uniqe VSC-id.
```

Comment:
In general, I think it is best to avoid the term "project" / "project account", because it is unnecessary and carries the risk of additional confusion.

```suggestion
For every job that gets submitted, the name of a credit account needs to be specified.
If you for example have been granted introduction credits, the corresponding credit
account will be named ``intro_vscxxxxx`` (with ``vscxxxxx`` referring to your VSC username).

Submitting a batch job can then look as follows::
```